src/lib/utils/command.ts

```diff
@@ -1,4 +1,9 @@
 export enum UtilsCommand {
   parseQRCodeString = "utils.parse_qr_code_string",
   tryParseDSKFromQRCodeString = "utils.try_parse_dsk_from_qr_code_string",
+  num2hex = "utils.num2hex",
+  formatId = "utils.format_id",
+  buffer2hex = "utils.buffer2hex",
+  getEnumMemberName = "utils.get_enum_member_name",
```

Comment:
I'm a bit torn on this one, as it's essentially ``enum[value] ?? `unknown (${value})` ``

Reply:
discussed offline, make a note recommending implementing those locally.